include/mme-app/procedureStats.h

```diff
@@ -87,6 +87,9 @@ namespace mme
                 static int num_of_abort_ded_activation;
                 static int num_of_ded_act_complete;
                 static int num_of_create_bearer_proc_complete;
+                static int num_of_identification_req_sent;
```

Comment:
You are planning to support Nucleus MME as old MME or new MME ? or both ?

Reply:
We are planning to support Nucleus MME as both.